website_product_configurator/static/src/js/config_form.esm.js

```diff
@@ -0,0 +1,629 @@
+/** @odoo-module **/
+
+import Dialog from "@web/legacy/js/core/dialog";
```

Comment:
nitpicking / not-blocking

This legacy `Dialog` is deprecated, and completely removed in `18`.

IMO we should migrate to the new `@web/core/dialog/dialog`.